src/confcom/samples/certs/create_certchain.sh

```diff
@@ -42,7 +44,7 @@ chmod 400 $RootPath/intermediateCA/private/intermediate.key.pem
 openssl req -config openssl_intermediate.cnf -key $RootPath/intermediateCA/private/intermediate.key.pem -new -sha256 -out $RootPath/intermediateCA/certs/intermediate.csr.pem -subj "/C=US/ST=Georgia/L=Atlanta/O=Microsoft/OU=ACCCT/CN=Intermediate CA"
 
 # sign intermediate cert with root
-openssl ca -config openssl_root.cnf -extensions v3_intermediate_ca -days 3650 -notext -md sha256 -in $RootPath/intermediateCA/certs/intermediate.csr.pem -out $RootPath/intermediateCA/certs/intermediate.cert.pem
+openssl ca -config openssl_root.cnf -extensions v3_intermediate_ca -days 3650 -notext -md sha256 -in $RootPath/intermediateCA/certs/intermediate.csr.pem -out $RootPath/intermediateCA/certs/intermediate.cert.pem -batch
```

Comment:
what does -batch do?

Reply:
it makes it so there's no prompt for signing

Reply:
Do you specify that somewhere in the readme?

Reply:
related: could it silently overwrite an existing file?

Reply:
> Do you specify that somewhere in the readme?

I had it in a readme that there were prompts, I'm taking it out now.



> related: could it silently overwrite an existing file / if so, would that be ok?

it can overwrite some of the files but not some of the .pem files. Do you think it's sufficient to put a note in the readme that consecutive runs won't overwrite completely?

Reply:
Yep, note in readme sounds good to me, ty!